index.html

```diff
@@ -40,11 +39,163 @@ <h3 class = "getintouch">
      <li><a class = "social-media"href="#"><img src="./images/medium.png" alt="medium"></a> </li>
      <li><a class = "social-media" href="#"><img src="./images/github.png" alt="github"></a></li>
      <li><a class = "social-media"href="#"><img src="./images/anglelist.png"alt="anglelist"></a></li>
-
+ </ul>
+ </div>
+</section>
+<section class="section">
+<section class="second-section">
+  <div class="section-2">
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio.png" alt="picture" >
+    </div>
+    <div>
+      <header class="proj-title">Tonic</header>
```

Comment:
_**THIS COMMENT IS MARKED [OPTIONAL] BECAUSE OF THE AMBIGUOUS INSTRUCTION OF THE PREVIOUS REVIEWER BUT I HIGHLY ADVISE YOU TO MAKE THE CHANGES AS THEY'RE PROJECT REQUIREMENTS**_
- [ ] [OPTIONAL] I'd like to heavily apologize once again as the previous reviewer meant you use a **_heading tag_** (for your case `h2-h6`) and not a `header` tag (as the header tag is specifically reserved for the header of your page). So please fix that. ✔